book/07-git-tools/sections/searching.asc

```diff
@@ -8,7 +8,7 @@ We'll go through a few of them.
 [[_git_grep]]
 ==== Git Grep
 
-Git ships with a command called `grep` that allows you to easily search through any committed tree or the working directory for a string or regular expression.
+Git ships with a command called `grep` that allows you to easily search through any committed tree, the working directory or even the index for a string or regular expression.
```

Comment:
I understand the Oxford comma is a bit controversial, but for consistency can you use it here?